frontend/src/Page/BikeGallery/BikeGallery.tsx

```diff
@@ -21,7 +21,7 @@ export default function BikeGallery(props: Props) {
                 p: 2
             }}>
                 <Typography variant="h4" component="h4" fontWeight={"bold"}>Your Bikes:</Typography>
-                <Button variant={"contained"} onClick={()=>navigate("/bikes/edit-form")}>
+                <Button variant={"contained"} onClick={()=>navigate("/bikes/add-bike")}>
```

Comment:
You could use a Link instead and style it appropriately, then you wouldn't need to call an anonymous function

Reply:
I dont do styling :-D
I let MUI do everything